cmd/homi/common/utils.go

```diff
@@ -73,6 +76,34 @@ func GenerateKeys(num int) (keys []*ecdsa.PrivateKey, nodekeys []string, addrs [
 	return keys, nodekeys, addrs
 }
 
+func GenerateKeysFromMnemonic(num int, mnemonic string) (keys []*ecdsa.PrivateKey, nodekeys []string, addrs []common.Address) {
```

Comment:
Can you add an option that can specify the coin type instead of 60 (Ethereum path)?
Recently, we recommend using Ethereum's path for our ecosystem wallets to provide the same address for users. That is our strategy for providing usability and other EVM chains are adopting the same approach, but strictly it is not the correct way of applying [BIP-44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki). Since Klaytn registered `8217` as a coin type on [SLIP-44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md). Some wallet builders may want to use the correct path, 8217.

Reply:
@aidan-kwon  How about taking the full path as an input so users can choose arbitrary paths? (`--mnemonic-path m/44'/60'/0'/0/` and alias `--mnemonic-path eth`)
In this case, which do you think the default path should be? eth or klay?

Reply:
Option for full path is a good way also. For default option, `eth` is better because Kaikas is using ethereum path now.